### ISSUE TITLE
Context broker is not notified about results

### ISSUE THREAD

**Issue by HoucemKacem** [smartfog/fogflow#221]
Hi again, 

Related to [this](https://github.com/smartfog/fogflow/issues/220) closed issue, I got all components up and running.

When an update occurs in the context broker, a task instance is triggered in Fogflow but the context broker does not receive any notification.
![Screenshot from 2021-03-19 12-20-06](https://user-images.githubusercontent.com/29146724/111773168-16044e00-88ae-11eb-827a-7d2b60974787.png)
I created a subscription on `daresult` entity types as mentioned in the docs.

I noticed that the `urn:ngsi-ld:Vehicle:A106daresult` entity was not created, since i got 404 response when retrieving it from the thinBroker. 

Here are the logs of all components: 
```
abbitmq_1      | 2021-03-19 11:18:20.541 [info] <0.1281.0> accepting AMQP connection <0.1281.0> (172.19.0.1:37340 -> 172.19.0.3:5672)
rabbitmq_1      | 2021-03-19 11:18:20.544 [info] <0.1281.0> connection <0.1281.0> (172.19.0.1:37340 -> 172.19.0.3:5672): user 'admin' authenticated and granted access to vhost '/'
cloud_broker_1  | notification from other broker
master_1        | INFO: 2021/03/19 11:18:28 ===========RECEIVE CONTEXT AVAILABILITY=========
master_1        | INFO: 2021/03/19 11:18:28 ===========RECEIVE CONTEXT AVAILABILITY=========
master_1        | DEBUG: 2021/03/19 11:18:28 master.go:482: &{fba7ab2b-9965-41d1-ab2f-caaa2f97594d [{{[{https://uri.etsi.org/ngsi-ld/default-context/Vehicle false urn:ngsi-ld:Vehicle:A106 }] [] [] http://10.5.3.105:8070/ngsi10} {200 OK }}] {301 OK }}
master_1        | DEBUG: 2021/03/19 11:18:28 master.go:482: &{ba66b306-f0b8-4f1d-950e-293ae7032c20 [{{[{https://uri.etsi.org/ngsi-ld/default-context/Vehicle false urn:ngsi-ld:Vehicle:A106 }] [] [] http://10.5.3.105:8070/ngsi10} {200 OK }}] {301 OK }}
master_1        | INFO: 2021/03/19 11:18:28 ===========RECEIVE CONTEXT AVAILABILITY=========
master_1        | DEBUG: 2021/03/19 11:18:28 master.go:482: &{96255a1d-2e9e-4796-9030-13064802e4a1 [{{[{https://uri.etsi.org/ngsi-ld/default-context/Vehicle false urn:ngsi-ld:Vehicle:A106 }] [] [] http://10.5.3.105:8070/ngsi10} {200 OK }}] {301 OK }}
master_1        | INFO: 2021/03/19 11:18:28 ===========RECEIVE CONTEXT AVAILABILITY=========
master_1        | DEBUG: 2021/03/19 11:18:28 master.go:482: &{11b26ad6-a09f-4349-8589-6afe0061f81c [{{[{https://uri.etsi.org/ngsi-ld/default-context/Vehicle false urn:ngsi-ld:Vehicle:A106 }] [] [] http://10.5.3.105:8070/ngsi10} {200 OK }}] {301 OK }}
master_1        | INFO: 2021/03/19 11:18:28 ===========RECEIVE CONTEXT AVAILABILITY=========
master_1        | DEBUG: 2021/03/19 11:18:28 master.go:482: &{2e8c18d3-89c7-4bb1-b34d-e1596dc81198 [{{[{https://uri.etsi.org/ngsi-ld/default-context/Vehicle false urn:ngsi-ld:Vehicle:A106 }] [] [] http://10.5.3.105:8070/ngsi10} {200 OK }}] {301 OK }}
master_1        | INFO: 2021/03/19 11:18:28 ===========RECEIVE CONTEXT AVAILABILITY=========
master_1        | DEBUG: 2021/03/19 11:18:28 master.go:482: &{8fcf6fef-562d-48d0-bca1-d781e295a3da [{{[{https://uri.etsi.org/ngsi-ld/default-context/Vehicle false urn:ngsi-ld:Vehicle:A106 }] [] [] http://10.5.3.105:8070/ngsi10} {200 OK }}] {301 OK }}
master_1        | INFO: 2021/03/19 11:18:28 ===========RECEIVE CONTEXT AVAILABILITY=========
master_1        | DEBUG: 2021/03/19 11:18:28 master.go:482: &{55aea6fc-91ae-435c-8278-284b18639421 [{{[{https://uri.etsi.org/ngsi-ld/default-context/Vehicle false urn:ngsi-ld:Vehicle:A106 }] [] [] http://10.5.3.105:8070/ngsi10} {200 OK }}] {301 OK }}
master_1        | DEBUG: 2021/03/19 11:18:28 master.go:540: REGISTERATION OF ENTITY CONTEXT AVAILABILITY: {ID:urn:ngsi-ld:Vehicle:A106 Type:https://uri.etsi.org/ngsi-ld/default-context/Vehicle AttributesList:map[] MetadataList:map[] ProvidingApplication:http://10.5.3.105:8070/ngsi10}
master_1        | INFO: 2021/03/19 11:18:28 handle the change of stream availability
master_1        | INFO: 2021/03/19 11:18:28 fba7ab2b-9965-41d1-ab2f-caaa2f97594d UPDATE urn:ngsi-ld:Vehicle:A106
master_1        | INFO: 2021/03/19 11:18:28 received registration: &{ID:urn:ngsi-ld:Vehicle:A106 Type:https://uri.etsi.org/ngsi-ld/default-context/Vehicle AttributesList:map[] MetadataList:map[] ProvidingApplication:http://10.5.3.105:8070/ngsi10}
master_1        | DEBUG: 2021/03/19 11:18:28 taskMgr.go:859: ~~~~~~~ access the flow ff_test.main, &{Intent:0xc00045c8f0 UniqueKeys:map[] Subscriptions:map[2e8c18d3-89c7-4bb1-b34d-e1596dc81198:0xc000063380] ExecutionPlan:map[] DeploymentPlan:map[]} ~~~~~~~~~~~~~~~~~
master_1        | DEBUG: 2021/03/19 11:18:28 taskMgr.go:868: nothing is triggered!!!
master_1        | DEBUG: 2021/03/19 11:18:28 master.go:540: REGISTERATION OF ENTITY CONTEXT AVAILABILITY: {ID:urn:ngsi-ld:Vehicle:A106 Type:https://uri.etsi.org/ngsi-ld/default-context/Vehicle AttributesList:map[] MetadataList:map[] ProvidingApplication:http://10.5.3.105:8070/ngsi10}
master_1        | INFO: 2021/03/19 11:18:28 handle the change of stream availability
master_1        | INFO: 2021/03/19 11:18:28 ba66b306-f0b8-4f1d-950e-293ae7032c20 UPDATE urn:ngsi-ld:Vehicle:A106
master_1        | INFO: 2021/03/19 11:18:28 received registration: &{ID:urn:ngsi-ld:Vehicle:A106 Type:https://uri.etsi.org/ngsi-ld/default-context/Vehicle AttributesList:map[] MetadataList:map[] ProvidingApplication:http://10.5.3.105:8070/ngsi10}
master_1        | DEBUG: 2021/03/19 11:18:28 taskMgr.go:859: ~~~~~~~ access the flow ff_test.main, &{Intent:0xc00045c8f0 UniqueKeys:map[] Subscriptions:map[2e8c18d3-89c7-4bb1-b34d-e1596dc81198:0xc000063380] ExecutionPlan:map[] DeploymentPlan:map[]} ~~~~~~~~~~~~~~~~~
master_1        | DEBUG: 2021/03/19 11:18:28 taskMgr.go:868: nothing is triggered!!!
master_1        | DEBUG: 2021/03/19 11:18:28 master.go:540: REGISTERATION OF ENTITY CONTEXT AVAILABILITY: {ID:urn:ngsi-ld:Vehicle:A106 Type:https://uri.etsi.org/ngsi-ld/default-context/Vehicle AttributesList:map[] MetadataList:map[] ProvidingApplication:http://10.5.3.105:8070/ngsi10}
master_1        | DEBUG: 2021/03/19 11:18:28 master.go:540: REGISTERATION OF ENTITY CONTEXT AVAILABILITY: {ID:urn:ngsi-ld:Vehicle:A106 Type:https://uri.etsi.org/ngsi-ld/default-context/Vehicle AttributesList:map[] MetadataList:map[] ProvidingApplication:http://10.5.3.105:8070/ngsi10}
master_1        | DEBUG: 2021/03/19 11:18:28 master.go:540: REGISTERATION OF ENTITY CONTEXT AVAILABILITY: {ID:urn:ngsi-ld:Vehicle:A106 Type:https://uri.etsi.org/ngsi-ld/default-context/Vehicle AttributesList:map[] MetadataList:map[] ProvidingApplication:http://10.5.3.105:8070/ngsi10}
master_1        | DEBUG: 2021/03/19 11:18:28 master.go:540: REGISTERATION OF ENTITY CONTEXT AVAILABILITY: {ID:urn:ngsi-ld:Vehicle:A106 Type:https://uri.etsi.org/ngsi-ld/default-context/Vehicle AttributesList:map[] MetadataList:map[] ProvidingApplication:http://10.5.3.105:8070/ngsi10}
master_1        | INFO: 2021/03/19 11:18:28 handle the change of stream availability
master_1        | INFO: 2021/03/19 11:18:28 55aea6fc-91ae-435c-8278-284b18639421 UPDATE urn:ngsi-ld:Vehicle:A106
master_1        | INFO: 2021/03/19 11:18:28 received registration: &{ID:urn:ngsi-ld:Vehicle:A106 Type:https://uri.etsi.org/ngsi-ld/default-context/Vehicle AttributesList:map[] MetadataList:map[] ProvidingApplication:http://10.5.3.105:8070/ngsi10}
master_1        | INFO: 2021/03/19 11:18:28 handle the change of stream availability
master_1        | DEBUG: 2021/03/19 11:18:28 taskMgr.go:859: ~~~~~~~ access the flow ff_test.main, &{Intent:0xc00045c8f0 UniqueKeys:map[] Subscriptions:map[2e8c18d3-89c7-4bb1-b34d-e1596dc81198:0xc000063380] ExecutionPlan:map[] DeploymentPlan:map[]} ~~~~~~~~~~~~~~~~~
master_1        | INFO: 2021/03/19 11:18:28 8fcf6fef-562d-48d0-bca1-d781e295a3da UPDATE urn:ngsi-ld:Vehicle:A106
master_1        | DEBUG: 2021/03/19 11:18:28 taskMgr.go:868: nothing is triggered!!!
master_1        | INFO: 2021/03/19 11:18:28 handle the change of stream availability
master_1        | INFO: 2021/03/19 11:18:28 11b26ad6-a09f-4349-8589-6afe0061f81c UPDATE urn:ngsi-ld:Vehicle:A106
master_1        | DEBUG: 2021/03/19 11:18:28 master.go:540: REGISTERATION OF ENTITY CONTEXT AVAILABILITY: {ID:urn:ngsi-ld:Vehicle:A106 Type:https://uri.etsi.org/ngsi-ld/default-context/Vehicle AttributesList:map[] MetadataList:map[] ProvidingApplication:http://10.5.3.105:8070/ngsi10}
master_1        | INFO: 2021/03/19 11:18:28 received registration: &{ID:urn:ngsi-ld:Vehicle:A106 Type:https://uri.etsi.org/ngsi-ld/default-context/Vehicle AttributesList:map[] MetadataList:map[] ProvidingApplication:http://10.5.3.105:8070/ngsi10}
master_1        | DEBUG: 2021/03/19 11:18:28 taskMgr.go:859: ~~~~~~~ access the flow ff_test.main, &{Intent:0xc00045c8f0 UniqueKeys:map[] Subscriptions:map[2e8c18d3-89c7-4bb1-b34d-e1596dc81198:0xc000063380] ExecutionPlan:map[] DeploymentPlan:map[]} ~~~~~~~~~~~~~~~~~
master_1        | DEBUG: 2021/03/19 11:18:28 taskMgr.go:868: nothing is triggered!!!
master_1        | INFO: 2021/03/19 11:18:28 received registration: &{ID:urn:ngsi-ld:Vehicle:A106 Type:https://uri.etsi.org/ngsi-ld/default-context/Vehicle AttributesList:map[] MetadataList:map[] ProvidingApplication:http://10.5.3.105:8070/ngsi10}
master_1        | DEBUG: 2021/03/19 11:18:28 taskMgr.go:859: ~~~~~~~ access the flow ff_test.main, &{Intent:0xc00045c8f0 UniqueKeys:map[] Subscriptions:map[2e8c18d3-89c7-4bb1-b34d-e1596dc81198:0xc000063380] ExecutionPlan:map[] DeploymentPlan:map[]} ~~~~~~~~~~~~~~~~~
master_1        | DEBUG: 2021/03/19 11:18:28 taskMgr.go:868: nothing is triggered!!!
master_1        | INFO: 2021/03/19 11:18:28 handle the change of stream availability
master_1        | INFO: 2021/03/19 11:18:28 2e8c18d3-89c7-4bb1-b34d-e1596dc81198 UPDATE urn:ngsi-ld:Vehicle:A106
master_1        | INFO: 2021/03/19 11:18:28 handle the change of stream availability
master_1        | INFO: 2021/03/19 11:18:28 received registration: &{ID:urn:ngsi-ld:Vehicle:A106 Type:https://uri.etsi.org/ngsi-ld/default-context/Vehicle AttributesList:map[] MetadataList:map[] ProvidingApplication:http://10.5.3.105:8070/ngsi10}
master_1        | INFO: 2021/03/19 11:18:28 96255a1d-2e9e-4796-9030-13064802e4a1 UPDATE urn:ngsi-ld:Vehicle:A106
master_1        | DEBUG: 2021/03/19 11:18:28 taskMgr.go:859: ~~~~~~~ access the flow ff_test.main, &{Intent:0xc00045c8f0 UniqueKeys:map[] Subscriptions:map[2e8c18d3-89c7-4bb1-b34d-e1596dc81198:0xc000063380] ExecutionPlan:map[] DeploymentPlan:map[]} ~~~~~~~~~~~~~~~~~
master_1        | DEBUG: 2021/03/19 11:18:28 taskMgr.go:173: create new entity
master_1        | INFO: 2021/03/19 11:18:28 received registration: &{ID:urn:ngsi-ld:Vehicle:A106 Type:https://uri.etsi.org/ngsi-ld/default-context/Vehicle AttributesList:map[] MetadataList:map[] ProvidingApplication:http://10.5.3.105:8070/ngsi10}
master_1        | DEBUG: 2021/03/19 11:18:28 taskMgr.go:489: unique key table map[Vehicle-EntityID:[urn:ngsi-ld:Vehicle:A106]]
master_1        | INFO: 2021/03/19 11:18:28 input available
master_1        | DEBUG: 2021/03/19 11:18:28 taskMgr.go:505: group key = EntityID
master_1        | DEBUG: 2021/03/19 11:18:28 taskMgr.go:524: info map[Vehicle-EntityID:urn:ngsi-ld:Vehicle:A106]
master_1        | DEBUG: 2021/03/19 11:18:28 taskMgr.go:102: group k: Vehicle-EntityID, v: urn:ngsi-ld:Vehicle:A106
master_1        | INFO: 2021/03/19 11:18:28 # hash = 741134294
master_1        | DEBUG: 2021/03/19 11:18:28 taskMgr.go:102: group k: Vehicle-EntityID, v: urn:ngsi-ld:Vehicle:A106
master_1        | DEBUG: 2021/03/19 11:18:28 taskMgr.go:555: SELECTOR {EntityType:Vehicle SelectedAttributes:[] GroupBy:EntityID Scoped:false}
master_1        | DEBUG: 2021/03/19 11:18:28 taskMgr.go:556: REGISTRATIONS map[urn:ngsi-ld:Vehicle:A106:0xc00029c680]
master_1        | DEBUG: 2021/03/19 11:18:28 taskMgr.go:557: UPDATED ENTITY urn:ngsi-ld:Vehicle:A106
master_1        | DEBUG: 2021/03/19 11:18:28 taskMgr.go:584: restriction map[EntityID:urn:ngsi-ld:Vehicle:A106]
master_1        | DEBUG: 2021/03/19 11:18:28 taskMgr.go:598: ENTITY REGISTRATION &{ID:urn:ngsi-ld:Vehicle:A106 Type:https://uri.etsi.org/ngsi-ld/default-context/Vehicle AttributesList:map[] MetadataList:map[] ProvidingApplication:http://10.5.3.105:8070/ngsi10}
master_1        | DEBUG: 2021/03/19 11:18:28 taskMgr.go:599: received input ENTITY {ID:urn:ngsi-ld:Vehicle:A106 Type:https://uri.etsi.org/ngsi-ld/default-context/Vehicle AttributeList:[] Location:{Latitude:0 Longitude:0}}
master_1        | DEBUG: 2021/03/19 11:18:28 taskMgr.go:102: group k: Vehicle-EntityID, v: urn:ngsi-ld:Vehicle:A106
master_1        | DEBUG: 2021/03/19 11:18:28 taskMgr.go:306: new task
master_1        | DEBUG: 2021/03/19 11:18:28 taskMgr.go:307: {Task.ff_test.main.741134294 main testing_overspeed  scheduled [{urn:ngsi-ld:Vehicle:A106 https://uri.etsi.org/ngsi-ld/default-context/Vehicle [] {0 0}}] [0xc0000fe000]}
master_1        | DEBUG: 2021/03/19 11:18:28 taskMgr.go:308: hashID 741134294, taskID Task.ff_test.main.741134294
master_1        | INFO: 2021/03/19 11:18:28 add task {ID:Task.ff_test.main.741134294 ServiceName:ff_test TaskName:main OperatorName:testing_overspeed TaskType: FunctionCode: DockerImage: Parameters:[] WorkerID: IsExclusive:false PriorityLevel:0 Status:scheduled Inputs:[{Type:https://uri.etsi.org/ngsi-ld/default-context/Vehicle ID:urn:ngsi-ld:Vehicle:A106 AttributeList:[]}] Outputs:[{Type:Out StreamID:Out.741134294.1 Annotations:[]}]}
master_1        | DEBUG: 2021/03/19 11:18:28 master.go:790: points: [{Latitude:0 Longitude:0}]
master_1        | INFO: 2021/03/19 11:18:28 check worker &{WID:Worker.001 PLocation:{Latitude:35.692221 Longitude:139.709059} GeohashID: OSType:linux HWType:amd64 Capacity:8 Workload:0 CAdvisorPort:0 EdgeAddress:}
master_1        | INFO: 2021/03/19 11:18:28 closest worker  Worker.001  with the closest distance  0
master_1        | INFO: 2021/03/19 11:18:28 select a suitable image to execute on the selected worker
master_1        | DEBUG: 2021/03/19 11:18:28 master.go:737: {testing_overspeed fogflow/overspeedvd latest X86 Linux false}
master_1        | DEBUG: 2021/03/19 11:18:28 master.go:738: &{Worker.001 {35.692221 139.709059}  linux amd64 8 0 0 }
master_1        | DEBUG: 2021/03/19 11:18:28 master.go:759: fogflow/overspeedvd:latest
master_1        | INFO: 2021/03/19 11:18:28 TASK INSTANCE TO BE DEPLOYED
master_1        | INFO: 2021/03/19 11:18:28 {Task.ff_test.main.741134294 ff_test main testing_overspeed   fogflow/overspeedvd:latest [] Worker.001 false 0 scheduled [{https://uri.etsi.org/ngsi-ld/default-context/Vehicle urn:ngsi-ld:Vehicle:A106 []}] [{Out Out.741134294.1 []}]}
master_1        | INFO: 2021/03/19 11:18:28 {ADD_TASK Worker.001. Master.001 {Task.ff_test.main.741134294 ff_test main testing_overspeed   fogflow/overspeedvd:latest [] Worker.001 false 0 scheduled [{https://uri.etsi.org/ngsi-ld/default-context/Vehicle urn:ngsi-ld:Vehicle:A106 []}] [{Out Out.741134294.1 []}]}}
master_1        | DEBUG: 2021/03/19 11:18:28 taskMgr.go:436: ==UPDATE DEPLOYMENT PLAN== &{Intent:0xc00045c8f0 UniqueKeys:map[Vehicle-EntityID:[urn:ngsi-ld:Vehicle:A106]] Subscriptions:map[2e8c18d3-89c7-4bb1-b34d-e1596dc81198:0xc000063380] ExecutionPlan:map[741134294:0xc000328580] DeploymentPlan:map[Task.ff_test.main.741134294:0xc00021e0f0]}
master_1        | DEBUG: 2021/03/19 11:18:28 taskMgr.go:859: ~~~~~~~ access the flow ff_test.main, &{Intent:0xc00045c8f0 UniqueKeys:map[Vehicle-EntityID:[urn:ngsi-ld:Vehicle:A106]] Subscriptions:map[2e8c18d3-89c7-4bb1-b34d-e1596dc81198:0xc000063380] ExecutionPlan:map[741134294:0xc000328580] DeploymentPlan:map[Task.ff_test.main.741134294:0xc00021e0f0]} ~~~~~~~~~~~~~~~~~
master_1        | DEBUG: 2021/03/19 11:18:28 taskMgr.go:868: nothing is triggered!!!
rabbitmq_1      | 2021-03-19 11:18:28.028 [info] <0.1298.0> accepting AMQP connection <0.1298.0> (172.19.0.1:37438 -> 172.19.0.3:5672)
rabbitmq_1      | 2021-03-19 11:18:28.030 [info] <0.1298.0> connection <0.1298.0> (172.19.0.1:37438 -> 172.19.0.3:5672): user 'admin' authenticated and granted access to vhost '/'
cloud_worker_1  | INFO: 2021/03/19 11:18:28 execute task  Task.ff_test.main.741134294  with operation fogflow/overspeedvd:latest
cloud_worker_1  | INFO: 2021/03/19 11:18:28 task configuration {ID:Task.ff_test.main.741134294 ServiceName:ff_test TaskName:main OperatorName:testing_overspeed TaskType: FunctionCode: DockerImage:fogflow/overspeedvd:latest Parameters:[] WorkerID:Worker.001 IsExclusive:false PriorityLevel:0 Status:scheduled Inputs:[{Type:https://uri.etsi.org/ngsi-ld/default-context/Vehicle ID:urn:ngsi-ld:Vehicle:A106 AttributeList:[]}] Outputs:[{Type:Out StreamID:Out.741134294.1 Annotations:[]}]}
cloud_worker_1  | INFO: 2021/03/19 11:18:28 task runnable =  true
cloud_worker_1  | =========This is task============
cloud_worker_1  | &{Task.ff_test.main.741134294 ff_test main testing_overspeed   fogflow/overspeedvd:latest [] Worker.001 false 0 running [{https://uri.etsi.org/ngsi-ld/default-context/Vehicle urn:ngsi-ld:Vehicle:A106 []}] [{Out Out.741134294.1 []}]}
cloud_worker_1  | INFO: 2021/03/19 11:18:28 to execute Task  Task.ff_test.main.741134294  to perform Operation  fogflow/overspeedvd:latest
rabbitmq_1      | 2021-03-19 11:18:28.042 [info] <0.1311.0> accepting AMQP connection <0.1311.0> (172.19.0.1:37442 -> 172.19.0.3:5672)
cloud_worker_1  | INFO: 2021/03/19 11:18:28 operator image fogflow/overspeedvd:latest exists locally
rabbitmq_1      | 2021-03-19 11:18:28.047 [info] <0.1311.0> connection <0.1311.0> (172.19.0.1:37442 -> 172.19.0.3:5672): user 'admin' authenticated and granted access to vhost '/'
cloud_worker_1  | INFO: 2021/03/19 11:18:28  task Task.ff_test.main.741134294  started within container = 8d83ce23ae935f7d85cf485eb7768442bdde0281f537ce8733b964b1060011ca
cloud_worker_1  | INFO: 2021/03/19 11:18:28 subscribe its input streams
cloud_worker_1  | 200
cloud_worker_1  | &{https://uri.etsi.org/ngsi-ld/default-context/Vehicle urn:ngsi-ld:Vehicle:A106 []}
cloud_worker_1  | DEBUG: 2021/03/19 11:18:28 executor.go:593:  =========== issue the following subscription =========== {Id: Type:Subscription Name: Description: Entities:[{Type:https://uri.etsi.org/ngsi-ld/default-context/Vehicle IsPattern:false ID:urn:ngsi-ld:Vehicle:A106 IdPattern:}] WatchedAttributes:[] TimeInterval:0 Q: GeoQ:{Geometry: Coordinates: GeoRel: GeoProperty:} Csf: IsActive:false Notification:{Attributes:[] Format: Endpoint:{URI:http://10.5.3.105:43131/notifyContext Accept:} Status: TimeSent:0 LastNotification: LastFailure: LastSuccess:} Expires: Throttling:0 TemporalQ:{TimeRel: Time: EndTime: TimeProperty:} Status: Subscriber:{IsOrion:false IsInternal:false RequireReliability:false BrokerURL: Integration:false NotifyCache:[] LDNotifyCache:[]} CreatedAt: ModifiedAt:}
cloud_worker_1  | DEBUG: 2021/03/19 11:18:28 executor.go:396: ===========subID =  05b56b72-c769-4bed-aa55-f17239e29f6e
cloud_worker_1  | http://10.5.3.105:8070/ngsi10
cloud_worker_1  | 404
cloud_worker_1  | INFO: 2021/03/19 11:18:28 register this task
cloud_broker_1  | INFO: 2021/03/19 11:18:28 UPDATE  subID  05b56b72-c769-4bed-aa55-f17239e29f6e
cloud_broker_1  | INFO: 2021/03/19 11:18:28 ===>  {https://uri.etsi.org/ngsi-ld/default-context/Vehicle false urn:ngsi-ld:Vehicle:A106 }  ,  05b56b72-c769-4bed-aa55-f17239e29f6e
cloud_broker_1  | INFO: 2021/03/19 11:18:28 http://10.5.3.105:8070/ngsi10 ,  http://10.5.3.105:8070/ngsi10
cloud_broker_1  | INFO: 2021/03/19 11:18:28 TO ngsi10 subscription,  05b56b72-c769-4bed-aa55-f17239e29f6e
cloud_broker_1  | INFO: 2021/03/19 11:18:28 entity list: [{Type:https://uri.etsi.org/ngsi-ld/default-context/Vehicle IsPattern:false ID:urn:ngsi-ld:Vehicle:A106 IdPattern:}]
cloud_broker_1  | false
cloud_broker_1  | INFO: 2021/03/19 11:18:28 NOTIFY:  http://10.5.3.105:43131/notifyContext
master_1        | INFO: 2021/03/19 11:18:28 ===========RECEIVE CONTEXT AVAILABILITY=========
master_1        | DEBUG: 2021/03/19 11:18:28 master.go:482: &{55aea6fc-91ae-435c-8278-284b18639421 [{{[{Task false Task.ff_test.main.741134294 }] [] [] http://10.5.3.105:8070/ngsi10} {200 OK }}] {301 OK }}
master_1        | DEBUG: 2021/03/19 11:18:28 master.go:540: REGISTERATION OF ENTITY CONTEXT AVAILABILITY: {ID:Task.ff_test.main.741134294 Type:Task AttributesList:map[id:{Name:id Type:string IsDomain:false Metadata:[]} operator:{Name:operator Type:string IsDomain:false Metadata:[]} port:{Name:port Type:string IsDomain:false Metadata:[]} service:{Name:service Type:string IsDomain:false Metadata:[]} status:{Name:status Type:string IsDomain:false Metadata:[]} task:{Name:task Type:string IsDomain:false Metadata:[]} worker:{Name:worker Type:string IsDomain:false Metadata:[]}] MetadataList:map[topology:{Name:topology Type:string Value:ff_test} worker:{Name:worker Type:string Value:Worker.001}] ProvidingApplication:http://10.5.3.105:8070/ngsi10}
master_1        | INFO: 2021/03/19 11:18:28 handle the change of stream availability
master_1        | INFO: 2021/03/19 11:18:28 55aea6fc-91ae-435c-8278-284b18639421 UPDATE Task.ff_test.main.741134294
master_1        | INFO: 2021/03/19 11:18:28 received registration: &{ID:Task.ff_test.main.741134294 Type:Task AttributesList:map[id:{Name:id Type:string IsDomain:false Metadata:[]} operator:{Name:operator Type:string IsDomain:false Metadata:[]} port:{Name:port Type:string IsDomain:false Metadata:[]} service:{Name:service Type:string IsDomain:false Metadata:[]} status:{Name:status Type:string IsDomain:false Metadata:[]} task:{Name:task Type:string IsDomain:false Metadata:[]} worker:{Name:worker Type:string IsDomain:false Metadata:[]}] MetadataList:map[topology:{Name:topology Type:string Value:ff_test} worker:{Name:worker Type:string Value:Worker.001}] ProvidingApplication:http://10.5.3.105:8070/ngsi10}
master_1        | DEBUG: 2021/03/19 11:18:28 taskMgr.go:859: ~~~~~~~ access the flow ff_test.main, &{Intent:0xc00045c8f0 UniqueKeys:map[Vehicle-EntityID:[urn:ngsi-ld:Vehicle:A106]] Subscriptions:map[2e8c18d3-89c7-4bb1-b34d-e1596dc81198:0xc000063380] ExecutionPlan:map[741134294:0xc000328580] DeploymentPlan:map[Task.ff_test.main.741134294:0xc00021e0f0]} ~~~~~~~~~~~~~~~~~
master_1        | DEBUG: 2021/03/19 11:18:28 taskMgr.go:868: nothing is triggered!!!
rabbitmq_1      | 2021-03-19 11:18:30.540 [info] <0.1328.0> accepting AMQP connection <0.1328.0> (172.19.0.1:37462 -> 172.19.0.3:5672)
rabbitmq_1      | 2021-03-19 11:18:30.544 [info] <0.1328.0> connection <0.1328.0> (172.19.0.1:37462 -> 172.19.0.3:5672): user 'admin' authenticated and granted access to vhost '/'
rabbitmq_1      | 2021-03-19 11:18:40.540 [info] <0.1345.0> accepting AMQP connection <0.1345.0> (172.19.0.1:37472 -> 172.19.0.3:5672)
rabbitmq_1      | 2021-03-19 11:18:40.543 [info] <0.1345.0> connection <0.1345.0> (172.19.0.1:37472 -> 172.19.0.3:5672): user 'admin' authenticated and granted access to vhost '/'
```

Last thing, for new updates from context broker, I noticed that no task instance was triggered and the logs shows:
```
rabbitmq_1      | 2021-03-19 11:32:20.540 [info] <0.2750.0> accepting AMQP connection <0.2750.0> (172.19.0.1:38432 -> 172.19.0.3:5672)
rabbitmq_1      | 2021-03-19 11:32:20.541 [info] <0.2750.0> connection <0.2750.0> (172.19.0.1:38432 -> 172.19.0.3:5672): user 'admin' authenticated and granted access to vhost '/'
cloud_broker_1  | notification from other broker
rabbitmq_1      | 2021-03-19 11:32:30.541 [info] <0.2767.0> accepting AMQP connection <0.2767.0> (172.19.0.1:38442 -> 172.19.0.3:5672)
rabbitmq_1      | 2021-03-19 11:32:30.546 [info] <0.2767.0> connection <0.2767.0> (172.19.0.1:38442 -> 172.19.0.3:5672): user 'admin' authenticated and granted access to vhost '/'
cloud_broker_1  | DEBUG: 2021/03/19 11:32:30 thinBroker.go:2746: resend notify:   1
cloud_broker_1  | false
cloud_broker_1  | INFO: 2021/03/19 11:32:30 NOTIFY:  http://10.5.3.105:43131/notifyContext
rabbitmq_1      | 2021-03-19 11:32:40.541 [info] <0.2784.0> accepting AMQP connection <0.2784.0> (172.19.0.1:38452 -> 172.19.0.3:5672)
rabbitmq_1      | 2021-03-19 11:32:40.546 [info] <0.2784.0> connection <0.2784.0> (172.19.0.1:38452 -> 172.19.0.3:5672): user 'admin' authenticated and granted access to vhost '/'
    
```
Any ideas about why the context broker is not getting notified about `daresult` entities ?

I appreciate your help.

**Comment by Neeraj-Nec** [comment]
Hi HoucemKacem 
As i am understanding according to the log you getting problem in  the step no 7 if you are following  this   https://fogflow.readthedocs.io/en/latest/scorpioIntegration.html  for  Document .
It will be easy for me  to identify the problem if you will provide the docker logs of running task.

**Comment by HoucemKacem** [comment]
Yes it's the step 7 in the document.
I've already put the logs of all services when the task is triggered until it's end.  
Can you please tell me what additional logs can I provide ?

**Comment by Neeraj-Nec** [comment]
Yes you have provided all the logs of FogFlow system .  But i am talking about task that you have mention in the screen shot . 
This is also a docker image . If you have only cloud environment then you can see this running image in the same environment .

**Comment by HoucemKacem** [comment]
Indeed, here are the logs of fogflow/overspeedvd:
```
myArgs:  [ '-o' ]
======== OPERATION MODE===========
[{"brokerURL":"http://10.5.3.105:8070/ngsi10","command":"CONNECT_BROKER"},{"command":"SET_REFERENCE","url":"http://10.5.3.105:43131"},{"command":"SET_OUTPUTS","id":"Out.741134294.1","type":"Out"}]
handle the initial admin configuration [{"brokerURL":"http://10.5.3.105:8070/ngsi10","command":"CONNECT_BROKER"},{"command":"SET_REFERENCE","url":"http://10.5.3.105:43131"},{"command":"SET_OUTPUTS","id":"Out.741134294.1","type":"Out"}]
{
  brokerURL: 'http://10.5.3.105:8070/ngsi10',
  command: 'CONNECT_BROKER'
}
http://10.5.3.105:8070
connected to broker http://10.5.3.105:8070/ngsi10
handle next command
{ command: 'SET_REFERENCE', url: 'http://10.5.3.105:43131' }
your application can subscribe addtional inputs under the reference URL:  http://10.5.3.105:43131
handle next command
{ command: 'SET_OUTPUTS', id: 'Out.741134294.1', type: 'Out' }
output has been set:  { command: 'SET_OUTPUTS', id: 'Out.741134294.1', type: 'Out' }
handle next command
{"time":"2021-03-19T11:18:28.604Z","lvl":"INFO","msg":"Starting NGSI Agent listening on port [43131]"}
start to receive input data streams via a listening port

```

**Comment by Neeraj-Nec** [comment]
Thank you  HoucemKacem  
It seems that FogFlow task is not getting notification from FogFlow broker. 
Let me check the problem .

**Comment by showersky** [comment]
Hi, 

I have looked into the log you shared. I think the problem is still with your firewall setting. Your task is expected to receive the notification at this Endpoint:{URI:http://10.5.3.105:43131/notifyContext, but the port 43131 is also blocked by your firewall so that your task could not receive the notification. 

Since this port for the task is decided on the fly, it is not possible to specify to your firewall in advance. If possible, please allow all the local connections with a rule in your firewall.

**Comment by HoucemKacem** [comment]
I disabled the firewall and now the context broker receives the expected notifications.
Thanks for your help !
